From 153262b3f293a0e75af2cee0970eb8fb99e28f75 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 8 May 2018 16:50:32 +0800
Subject: [PATCH] regulator: rk808: make rk8xx_is_enabled_wmsk_regmap static

The function rk8xx_is_enabled_wmsk_regmap is local to the source
and does not need to be in global scope, so make it static.

drivers/regulator/rk808-regulator.c:714:5: warning: no previous prototype for 'rk8xx_is_enabled_wmsk_regmap' [-Wmissing-prototypes]

Change-Id: Iad140bc1acba548caee1a333a9ed1f846ec06b27
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/regulator/rk808-regulator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/regulator/rk808-regulator.c b/drivers/regulator/rk808-regulator.c
index 9924740c95f8..0c9f4404c7e8 100644
--- a/drivers/regulator/rk808-regulator.c
+++ b/drivers/regulator/rk808-regulator.c
@@ -711,7 +711,7 @@ static unsigned int rk8xx_get_mode(struct regulator_dev *rdev)
 		return REGULATOR_MODE_NORMAL;
 }
 
-int rk8xx_is_enabled_wmsk_regmap(struct regulator_dev *rdev)
+static int rk8xx_is_enabled_wmsk_regmap(struct regulator_dev *rdev)
 {
 	unsigned int val;
 	int ret;
-- 
2.35.3

